src/index.ts

```diff
@@ -132,7 +132,9 @@ export async function isAvailable() {
       return false;
     } else if (
       err.code &&
-      (err.code === 'ENOTFOUND' || err.code === 'ENOENT')
+      (err.code === 'ENOTFOUND' ||
```

Comment:
is there an exhaustive list of these codes anywhere?  Suddenly I'm worried we're missing others :)

Reply:
we talked about this a bit in chat, there's probably not one canonical list that will work well for all OS (it's one of those, keep an eye and try to get it right sort of things).